infrastructure-playbooks/purge-cluster.yml

```diff
@@ -88,8 +88,15 @@
     - name: ensure cephfs mountpoint(s) are unmounted
       command: umount -a -t ceph
 
+    - name: check if rbdmap is still installed
+      command: which rbdmap
+      register: which_rbdmap
```

Comment:
change the name accordingly to the previous comment

Reply:
And done.